springerlink.md

```diff
@@ -1,7 +1,7 @@
 # SpringerLink Accessibility Conformance Report International Edition 
 (Based on VPAT® Version 2.4)
 
-**Report Date**: 10th June 2021  
+**Report Date**: 2nd August 2021  
 **Name of Product/Version**: SpringerLink _(we do not version our software)_  
 **Product Description**: SpringerLink provides researchers with access to scientific documents from journals, books, series, protocols, reference works and proceedings. This report refers entirely to the accessibility of all web documents published at the subdomains link.springer.com and rd.springer.com (both subdomains are referred to as link.springer.com throughout this report), including informational pages, and all web documents found at checkout.springer.com and order.springer.com.  
```

Comment:
Does 'collections' need to be added here?

Do the content types listed need to be capitalised as it has been done further down the document?

Reply:
yes to the former, no to the latter - but only because I don't think that stuff should have been capitalised in the first place. Have added collections to the scope declaration and also removed the exuberant capitalisation throughout the doc 🙈